src/index.js

```diff
@@ -17,38 +16,32 @@ class Aurelia {
 
   init(config) {
     this.config = config;
-    let bundle = new BundleCommand(program, this.config, this.logger);
-    let init = new InitCommand(program, this.config, this.logger);
-    let newCmd = new NewCommand(program, this.config, this.logger);
-    let generateCmd = new GenerateCommand(program, this.config, this.logger);
-
-    this.commands[bundle.commandId] = bundle;
-    this.commands[init.commandId] = init;
-    this.commands[newCmd.commandId] = newCmd;
-    this.commands[generateCmd.commandId] = generateCmd;
-
+    program.init(config);
+    program.register(BundleCommand);
```

Comment:
@joelcoxokc Is it possible to have it like: `program.register(BundleCommand, this.config, this.logger)` This way we/framework will have the control what to inject in the command.